### DISCUSSION TITLE
Improve Krunker Support

### DISCUSSION THREAD

**Discussion by unrealapex** [QuiteAFancyEmerald/InvisiProxy#41] [Ideas]
Currently, the time it takes to load krunker is high and it is not possible to connect to a live game. 

### Screenshots 
----
![Screenshot 2021-03-25 1 35 23 PM](https://user-images.githubusercontent.com/79121360/112540171-18175280-8d6f-11eb-839c-9a40798400b4.png)
![Screenshot 2021-03-25 1 35 07 PM](https://user-images.githubusercontent.com/79121360/112540194-1ea5ca00-8d6f-11eb-87ea-6f2c33fedc2a.png)

**Comment by QuiteAFancyEmerald** [comment]
#24
